x/authz/msgs.go

```diff
@@ -144,11 +145,11 @@ func (msg MsgRevoke) ValidateBasic() error {
 	}
 
 	if granter.Equals(grantee) {
-		return sdkerrors.Wrap(sdkerrors.ErrInvalidRequest, "granter and grantee cannot be same")
+		return sdkerrors.ErrInvalidRequest.Wrap("granter and grantee cannot be same")
```

Comment:
just a doubt, can we use `authz.ErrGranteeIsGranter` here?

Reply:
I don't see a reason not to. Changed 👍🏾

Reply:
Yes, we prob should. Let's update this @julienrbrt :)